packages/frontend/src/hooks/dashboard/useDashboardFiltersForExplore.ts

```diff
@@ -34,6 +34,14 @@ const useDashboardFiltersForExplore = (
                 ...dashboardFilters.metrics,
                 ...(dashboardTemporaryFilters?.metrics ?? []),
             ]),
+            tableCalculations: getDashboardFilterRulesForTile(
```

Comment:
This method does not exist, perhaps you mean `getDashboardFilterRulesForTileAndTables` ?

Reply:
same as above, i will correct this